js/api.js

```diff
@@ -29,14 +29,15 @@ export const searchPosts = async (query, sort = true) => {
   return results
 }
 
-export const getComments = (post, comment) =>
+export const getComments = (post, comment, sort = 'new') =>
```

Comment:
I feel like the default here should be the same as reddit's default.
It's called "Best" or "hot" on the site, not sure which one that corresponds to on the API, maybe it's named differently or you just don't pass a `sort` param  :thinking:

Reply:
I agree, the default is now confidence / best.